packages/calling/src/CallingClient/CallingClient.ts

```diff
@@ -491,6 +413,40 @@ export class CallingClient extends Eventing<CallingClientEventTypes> implements
   public getLines(): Record<string, ILine> {
     return this.lineDict;
   }
+
+  /**
+   * Retrieves call objects for all the active calls present in the client
+   */
+  public getActiveCalls(): Record<string, ICall[]> {
+    const activeCalls = {};
+    const calls = this.callManager.getActiveCalls();
+    Object.keys(this.lineDict).forEach((deviceId) => {
```

Comment:
Doesn't lineDict has lineId as a key
```suggestion
    Object.keys(this.lineDict).forEach((lineId) => {
```

Reply:
```suggestion
  Object.keys(calls).forEach((correlationId) => {
      const call = calls[correlationId];
      if (!Object.keys(activeCalls).includes(call.lineId)) {
        activeCalls[call.lineId]  = [];
      } 
      activeCalls[call.lineId].append(call);
      
    });
```

Reply:
Changed to the suggested changes to make the code more readable